### PR TITLE
docs(ootbc): add twilio webhook connector

### PR DESCRIPTION
**Pull request by Oleksiivanov**
## Description

Added Twilio webhook documentation

Related : https://github.com/camunda/product-hub/issues/1138

## When should this change go live?

- [ ] This change is not yet live and should not be merged until 11 July (after release [Twilio webhook connector](https://github.com/camunda/product-hub/issues/1138))
## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory
- [x] I have added changes to the main `/docs` directory (aka `/next/`)
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer

### CODE REVIEW

File: docs/components/connectors/out-of-the-box-connectors/twilio-webhook.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    ...
+  }
+  ...
+}
+```
+
+your correlation key settings will look like this:
+
+- **Correlation key (process)**: `=myCorrelationKey`
+- **Correlation key (payload)**: `=request.body.FromCountry`
```
Comment (bastiankoerber) [inline docs/components/connectors/out-of-the-box-connectors/twilio-webhook.md:84]: is this a valid use case? i would expect something like a message id

**Review by bastiankoerber — DISMISSED** [review]
nice documentation. 
I just would add better example for a correlation key

File: docs/components/connectors/out-of-the-box-connectors/twilio-webhook.md (hunk 1 above)
Comment (igpetrov) [reply docs/components/connectors/out-of-the-box-connectors/twilio-webhook.md:84]: Message ID is unknown at this point. Probably `Body` (text message) is better fit

**Review by igpetrov — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]
Technical adjustments in latest commit 👍

**Review by sbuettner — APPROVED** [review]

**Review by igpetrov — APPROVED** [review]

**Review by sbuettner — APPROVED** [review]

**Pull request merged**